src/rust/engine/fs/store/src/lib.rs

```diff
@@ -533,6 +533,21 @@ impl Store {
       .await
   }
 
+  ///
+  /// Ensures that the directory entries of the given DirectoryDigest is persisted to disk.
+  ///
+  /// TODO: By the end of #13112, usage of this method should be limited to the writing of cache
```

Comment:
These TODOs should probably link to the issue #

Reply:
They do link to the issue number... do you mean: with a url?

Reply:
@Eric-Arellano: Did you mean the `TODO(NNNN)` notation?

Reply:
Ah you're right. I could have sworn I saw one or two without the issue linked, but my eyes are deceiving me. 

I did also mean `TODO(#nnnn)` which our style guide says to use, which I think is for better grep searches. But that's busy-work, this is fine.